src/components/search-by-drug-orders/search-by-drug-orders.utils.ts

```diff
@@ -0,0 +1,116 @@
+import dayjs from "dayjs";
+
+import { composeJson } from "../../cohort-builder.utils";
+import { DrugOrderDetails } from "./../../types/index";
+
+export const getDescription = ({
+  selectedDrugs,
+  selectedCareSetting,
+  activeOnOrBefore,
+  activeOnOrAfter,
+  activatedOnOrBefore,
+  activatedOnOrAfter,
+}: DrugOrderDetails) => {
+  let description =
+    "Patients who taking " +
+    selectedDrugs
```

Comment:
Please make a different variable for this operation. Makes it easy to read code when looking at the first time for any update in the code in the future.